source/release/4.2.0/index.html.md

```diff
@@ -0,0 +1,710 @@
+
+#### oVirt Host Deploy
+
+ - [BZ 1471196](https://bugzilla.redhat.com/1471196) <b>Adding a new host in engine running on fedora 26 fails</b><br>
+ - [BZ 1473163](https://bugzilla.redhat.com/1473163) <b>Do not install collectd-virt plugin</b><br>
+
+#### ovirt-engine-dwh
+
+ - [BZ 1432916](https://bugzilla.redhat.com/1432916) <b>DWH collects incorrect statistics for shared vm disks.</b><br>
+
+#### oVirt Hosted Engine HA
+
+ - [BZ 1378310](https://bugzilla.redhat.com/1378310) <b>Continuously restarting ha-agent breaks failover</b><br>
+ - [BZ 1379250](https://bugzilla.redhat.com/1379250) <b>[TEXT] - A comment to vm.conf of the hosted engine</b><br>
+ - [BZ 1488333](https://bugzilla.redhat.com/1488333) <b>ovirt-ha-agent fails connecting ovirt-ha-broker</b><br>
+
+#### oVirt Engine Metrics
+
+ - [BZ 1471833](https://bugzilla.redhat.com/1471833) <b>[RFE]  Allow passing extra ansible opts to oVirt Metrics shell script</b><br>
+ - [BZ 1477083](https://bugzilla.redhat.com/1477083) <b>[RFE] Add cluster name as a field to the hosts and vm records</b><br>
+ - [BZ 1469104](https://bugzilla.redhat.com/1469104) <b>Remove host statistics collected from vdsm except for storage</b><br>
+ - [BZ 1469119](https://bugzilla.redhat.com/1469119) <b>Disable collectd virt plugin</b><br>
+ - [BZ 1468894](https://bugzilla.redhat.com/1468894) <b>warning message appears when running the metrics setup script regarding jinja2 templating</b><br>
+
+#### oVirt Hosted Engine Setup
+
+ - [BZ 1356425](https://bugzilla.redhat.com/1356425) <b>[TEXT] 'hosted-engine --vm-start' said it destroyed the VM</b><br>
+ - [BZ 1348225](https://bugzilla.redhat.com/1348225) <b>hosted-engine --deploy sorts iscsi luns incorrectly</b><br>
+
+#### oVirt Log collector
+
+ - [BZ 1488630](https://bugzilla.redhat.com/1488630) <b>03_06_0620_create_fence_agents_table.sql:60: ERROR:  null value in column "agent_user" violates not-null constraint</b><br>
+
+#### oVirt Provider OVN
+
+ - [BZ 1490104](https://bugzilla.redhat.com/1490104) <b>vdsm-tool does not start required services before configuring the host</b><br>
+ - [BZ 1447875](https://bugzilla.redhat.com/1447875) <b>Create a conf.d for ovirt-provider-ovn configuration</b><br>
+ - [BZ 1481985](https://bugzilla.redhat.com/1481985) <b>Token 'ServiceCatalog' returns incorrect URLs</b><br>
+ - [BZ 1454694](https://bugzilla.redhat.com/1454694) <b>ovirt-provider-ovn-central firewalld service is no longer needed</b><br>
+
+#### oVirt Release Package
+
+ - [BZ 1464527](https://bugzilla.redhat.com/1464527) <b>[centos-qemu-ev-release] GPG key not found</b><br>
+ - [BZ 1488189](https://bugzilla.redhat.com/1488189) <b>Enable centos-release-scl-rh on el7 x86_64</b><br>
+ - [BZ 1443965](https://bugzilla.redhat.com/1443965) <b>Libvirt is disabled on RHVH host</b><br>
+
+#### oVirt Engine Appliance
+
+ - [BZ 1406493](https://bugzilla.redhat.com/1406493) <b>Use livemedia-creator instead of image-tools</b><br>
+
+### No Doc Update
```

Comment:
What does "no doc update" mean?

Reply:
Developer decided that no special documentation is needed